### PR TITLE
Installation Guide Documentation

### PR DESCRIPTION
**Pull request by Jen-Lopez**
**Description:**

Created the Installation folder that contains the installation guides using docker and the guide on the website. Also updated the readme "How do I get set up" section with the corresponding guide links.

**Related issues and discussion:** [#3204](https://github.com/openfoodfacts/openfoodfacts-server/issues/3204)

### CODE REVIEW

File: installation/product-opener.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+Installation Instructions for Product Opener
+
+### Product Opener Database
+Product Opener is a software used to create and maintain the database of food products in open data. 
+It is released under the GNU Affero License, which is a free copyleft license. This allows Product Opener to be reused by other projects.
+
+### Before you Start
+
+Product opener is NOT yet released as open source. The instructions are for reference and for early testers of Product Opener. These instructions are very likely to change as the code will be restructured, made more independent of proprietary code, and specific Open Food Facts uses etc. When it is released, Product Opener will have much easier to use install scripts and instruction.
```
Comment (hangy) [inline installation/product-opener.md:11]: I think this paragraph from the wiki might be outdated. It's released under the AGPL and being developed as FOSS here on GitHub.

Comment (Jen-Lopez) [reply installation/product-opener.md]: Sorry about that we fixed the file. We'll also update the wiki to reflect these new changes.

File: installation/product-opener.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Product Opener Installation
+
+Product opener is released under the AGPL license and is being developed as [Free and Open Source](https://en.wikipedia.org/wiki/Free_and_open-source_software)  on Github.
+
+### Installation
+Installation Instructions for Product Opener
+[Product Opener Installation](https://en.wiki.openfoodfacts.org/Product_Opener/Installation)
```
Comment (stephanegigandet) [inline installation/product-opener.md:7]: This wiki page is very old and contained outdated installation instructions.

I replaced its content with a link to https://en.wiki.openfoodfacts.org/Infrastructure#Server_configuration which contains newer instructions (but not necessarily completely up to date either).

**Comment by Jen-Lopez** [comment]
There was a merge conflict with one of the files but it's resolved now.

**Comment by teolemon** [comment]
Thank you @Jen-Lopez ! merging

**Pull request merged**